person.rb

```diff
@@ -1,42 +1,15 @@
-class Nameable
-  def correct_name
-    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
-  end
-end
-
-class BaseDecorator < Nameable
-  def initialize(nameable)
-    super()
-    @nameable = nameable
-  end
-
-  def correct_name
-    @nameable.correct_name
-  end
-end
-
-class CapitalizeDecorator < BaseDecorator
-  def correct_name
-    @nameable.correct_name.capitalize
-  end
-end
-
-class TrimmerDecorator < BaseDecorator
-  def correct_name
-    str = @nameable.correct_name
-    str[0...10] if str.length >= 10
-  end
-end
+require './decorators/person/correct_name'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', _parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
+    @rentals = []
```

Comment:
- [ ] As mentioned above in `book.rb` let's kindly make sure we also have one-to-many relationship between `Person` and `Rental` by defining a method here as well with the suitable changes 👍🏻